pypeit/core/coadd2d.py

```diff
@@ -90,20 +91,26 @@ def optimal_weights(specobjs_list, slitid, objid, echelle=True):
 
     return rms_sn, weights, trace_stack, wave_stack
 
-def load_coadd2d_stacks(spec2d_files):
+def load_coadd2d_stacks(spec2d_files, det=1):
```

Comment:
need doc string

Reply:
Coming soon in the next PR when this stuff is finalized.